app/api/routes/login.py

```diff
@@ -1,8 +1,34 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, HTTPException, status
+from fastapi.security import OAuth2PasswordRequestForm
+from datetime import timedelta
+from app.models import *
+from app.api.dependencies.auth import *
+from app.api.services import user_service
+from app.api.dependencies.password_utils import *
+
 
 router = APIRouter()
 
-# make sure to remove this function
-@router.get("/example")
-def example():
-    return {"message": "this is an example"}
+user_service = user_service.UserService()
+@router.post("/login", response_model=Token)
+async def login_for_access_token(form_data: OAuth2PasswordRequestForm = Depends()):
+    user = authenticate_user(form_data.username, form_data.password)
+    if not user:
+        raise HTTPException(
+            status_code=status.HTTP_401_UNAUTHORIZED,
+            detail="Incorrect username or password",
+            headers={"WWW-Authenticate": "Bearer"},
+        )
+    access_token_expires = timedelta(minutes=ACCESS_TOKEN_EXPIRE_MINUTES)
+    access_token = create_access_token(
+        data={"sub": user["uuid"]}, expires_delta=access_token_expires
+    )
+    return {"access_token": access_token, "token_type": "bearer"}
+
+@router.post("/register", response_model=UserDTO)
+def create_admin_user(user: User):
```

Comment:
Register is for normal user, so the function name is not accurate. 
If we need to change that to admin,.we have another endpoint for that called change role.

And you need to check if the email is already registered